### DISCUSSION TITLE
Externe LED leuchtet nicht immer

### DISCUSSION THREAD

**Discussion by berlinermainzer** [jomjol/AI-on-the-edge-device#2493] [Q&A]
Hallo zusammen,

ich habe aufgrund von zu starken Reflexionen die Beleuchtung auf 2 WS2812B LEDs umgestellt wie unter https://jomjol.github.io/AI-on-the-edge-device-docs/External-LED/ beschrieben. Im Prinzip funktioniert das gut - Reflexionen sind quasi weg und die Erkennung der Ziffern klappt auch.
Allerdings werden die LEDs nicht immer eingeschaltet, wenn ein Foto für eine neue Runde gemacht wird. Ein Muster kann ich nicht erkennen, wann es geht oder nicht.
Ich hatte erst das Netzteil in Verdacht, aber auch mit einem 5V/2A Netzteil läuft es nicht zuverlässig.

Habt ihr eine Idee, was das Problem sein kann?

Danke & viele Grüße
Johannes

**Comment by caco3** [comment]
Kannst Du mal einen debug log über mehrere gute/schlechte Runden liefern?

Ich habe selber keine externe LEDs, aber ev. kann man etwas im Log sehen.
